compiler/rustc_middle/src/mir/pretty.rs

```diff
@@ -580,7 +580,7 @@ fn write_scope_tree(
             continue;
         }
 
-        let mut_str = if local_decl.mutability == Mutability::Mut { "mut " } else { "" };
+        let mut_str = if local_decl.mutability.is_mut() { "mut " } else { "" };
```

Comment:
ast::Mutability has a helper for this... maybe we should merge the three `Mutability` enums

Reply:
This _is_ actually `ast::Mutability`, I just forgor about the helper 😅